Examples/Benchmarks/VectorBenchmark.som

```diff
@@ -0,0 +1,26 @@
+"A benchmark for the Builtin Vector class"
+
+VectorBenchmark = Benchmark (
+
+    benchmark = (
+
+        "Create an Vector and append 1000 items to it"
+        | arr resultVec |
+        arr := Vector new.
+        1 to: 1000 do: [:i | arr append: i].
+
+        "Iterate over and create a new result vector to store even number"
+        resultVec := Vector new.
+        1 to: 1000 do: [:i |
+            ((arr at: i) % 2) = 0 ifTrue: [
```

Comment:
If we want to focus on the vector operations, maybe we just check that the value is not 0, to avoid introducing other work for the modulo operation?
With benchmarks, it's always ending up a question of what we want to emphasize.